src/widget_type.h

```diff
@@ -374,7 +374,8 @@ class NWidgetCore : public NWidgetResizeBase {
 
 	void SetDataTip(uint32_t widget_data, StringID tool_tip);
 	void SetStringTip(StringID string, StringID tool_tip) { this->SetDataTip(string, tool_tip); }
-	void SetSpriteTip(SpriteID sprite, StringID tool_tip) { this->SetDataTip(sprite, tool_tip); }
+	void SetSprite(SpriteID sprite);
+	void SetSpriteTip(SpriteID sprite, StringID tool_tip) { this->SetSprite(sprite); this->SetToolTip(tool_tip); }
```

Comment:
Not sure this should be a one-liner anymore

Reply:
I'll fix that up when doing `SetString` since `SetString` and `SetSprite` conflict with each other.